control/mpc_lateral_controller/src/mpc.cpp

```diff
@@ -297,7 +294,8 @@ bool MPC::getData(
   if (std::fabs(data->yaw_err) > m_admissible_yaw_error_rad) {
     RCLCPP_WARN_SKIPFIRST_THROTTLE(
       m_logger, *m_clock, duration, "yaw error is over limit. error = %f deg, limit %f deg",
-      RAD2DEG * data->yaw_err, RAD2DEG * m_admissible_yaw_error_rad);
+      tier4_autoware_utils::rad2deg(1) * data->yaw_err,
```

Comment:
Why don't use implement like this?
`tier4_autoware_utils::rad2deg(data->yaw_err)`

Reply:
thanks, I missed it when making a quick change.